src/iqm/iqm_client/iqm_client.py

```diff
@@ -97,6 +99,7 @@ def __init__(
         auth_server_url: Optional[str] = None,
         username: Optional[str] = None,
         password: Optional[str] = None,
+        api_variant: Optional[APIVariant] = None,
```

Comment:
```suggestion
        auth_server_url: str | None = None,
        username: str | None = None,
        password: str | None = None,
        api_variant: APIVariant | None = None,
```

It seems like Python folks are preferring the union pipe now. But I guess to make this work in Python 3.9 we should also do `from future import annotations` (this is a weird quirk).

Reply:
Yes, I decided to keep the old while still support Python 3.9